src/tests/ftest/launch.py

```diff
@@ -57,17 +57,16 @@
 LOG_FILE_FORMAT = "%(asctime)s %(levelname)-5s %(funcName)30s: %(message)s"
 MAX_CI_REPETITIONS = 10
 TEST_EXPECT_CORE_FILES = ["./harness/core_files.py"]
-PROVIDER_KEYS = OrderedDict(
-    [
-        ("cxi", "ofi+cxi"),
-        ("verbs", "ofi+verbs"),
-        ("ucx", "ucx+dc_x"),
-        ("tcp", "ofi+tcp;ofi_rxm"),
-        ("opx", "ofi+opx"),
-    ]
-)
+SUPPORTED_PROVIDERS = [
+    "ofi+cxi",
+    "ofi+verbs;ofi_rxm",
+    "ucx+dc_x",
+    "ofi+tcp;ofi_rxm",
+    "ofi+opx",
+]
 # Temporary pipeline-lib workaround until DAOS-13934 is implemented
 PROVIDER_ALIAS = {
+    "ofi+verbs": "ofi+verbs;ofi_rxm",
```

Comment:
Good change. Eventually we'll have a non-RXM verbs provider, so this is good preparation. :)